templates/index.html

```diff
@@ -2,15 +2,115 @@
+                <!-- Alert to user notifying of successful logout. -->
+                {% with alerts = get_flashed_messages(with_categories=true) %}
+                    {% if alerts %}
+                        {% for category, alert in alerts %}
+                            <div class="alert alert-{{ category }} mt-5">
+                                {{ alert }}
+                            </div>
+                        {% endfor %}
+                    {% endif %}
+                {% endwith %}
+            </div>
+        </div>
+        <div class="row greeting">
+            {% if 'username' in session %}
+            <div class="col-sm-12 text-center">
+                <p>Welcome, {{ session['username'] }}</p>
+            </div>
+            {% endif %}
+
+            <div class="col-sm-12 text-center">
+                <h2>Welcome to Chrysalis Coding!</h2>
+            </div>
+        </div>
+        <div class="row">
+            <div class="col-sm-12 text-center">
+                <p class="page-desc">
+                    At Chrysalis Coding, we provide a space where Women in Tech and advocates of this movement to
+                    detail their experiences of working within this industry. Enabling future generations with
+                    inspiration and learning to bring and adopt into their own experiences.
+                </p>
+            </div>
+            <div class="col-sm-12 text-center">
+                <p class="page-desc">
+                    This site was designed to promote the positives and the negatives of working as a Woman in Tech,
+                    or as an onlooker. It is designed to work on the go, and we all have mobile smartphones at our disposal,
+                    meaning you can read or add to the discussions in the Forum at any time, with the touch of a button right
+                    from your own pocket! Together, we can further promote Women in Tech, and empower the future!
+                </p>
+            </div>
+        </div>
+        <div class="row">
+            <div class="col-sm-12 brand-logo" aria-label="Brand Logo"></div>
+        </div>
+    </div>
+
+    <div class="container">
+        <div class="row card-row">
+            <div class="card home-card col-sm-12 col-md-6">
+                <div class="card-body text-center">
+                    {% if 'username' in session %}
+                    <h5 class="card-title hca-title">View our inspiration!</h5>
+                    <p class="card-text">
+                        We needed our own inspiration to start up this independent project of ours! like
+                        with any great idea, it started with a movement, or a spark from someone before us.
+                    </p>
+                    <p class="card-text hca-p2">
+                        Read about these inspiration Women in Tech. Those who came before and had to strive
+                        to prove themselves in a, then dominated, industry.
```

Comment:
This probably is missing the word 'male' in it (male-dominated). (Just checking out how to create comments here.. :) )